src/sql/workbench/contrib/notebook/browser/find/notebookFindDecorations.ts

```diff
@@ -15,20 +15,17 @@ import { NotebookRange } from 'sql/workbench/services/notebook/browser/notebookS
 
 export class NotebookFindDecorations implements IDisposable {
 
-	private _decorations: string[];
-	private _overviewRulerApproximateDecorations: string[];
-	private _findScopeDecorationId: string | null;
-	private _rangeHighlightDecorationId: string | null;
-	private _highlightedDecorationId: string | null;
+	private _decorations: string[] = [];
+	private _overviewRulerApproximateDecorations: string[] = [];
+	private _findScopeDecorationIds: string[] = [];
+	private _codeCellFindScopeDecorationIds: string[] = [];
+	private _rangeHighlightDecorationId: string | null = null;
```

Comment:
why do we use null instead of undefined here?

Reply:
I've followed the vscode style from their findDecorations class.